src/i_winmusic.c

```diff
@@ -453,7 +461,7 @@ static void UpdateVolume()
 
 static void I_WIN_SetMusicVolume(int volume)
 {
-    volume_factor = (float)volume / 15;
+    volume_factor = (float)volume * 8.0f / 100.0f;
```

Comment:
Why float?

Reply:
I was being overly cautious. It's not needed.